rational.c

```diff
@@ -420,8 +402,6 @@ nurat_s_alloc(VALUE klass)
     return nurat_s_new_internal(klass, ZERO, ONE);
 }
 
-#define rb_raise_zerodiv() rb_raise(rb_eZeroDivError, "divided by 0")
-
 #if 0
```

Comment:
Why not remove this disabled range?

Reply:
I remembered this just today, but I confirmed this was fixed by c225d3c9b52e by @mrkn . Thank you!